src/lv_interop.h

```diff
@@ -93,6 +93,20 @@ namespace grpc_labview
             return (T*)(rawBytes + byteOffset);
     #endif
         }
+
+        void* bytes(int byteOffset, int byteAlignment)
```

Comment:
The bytes methods above made assumptions about where the raw data of the array would start.  This assumption broke for an array of cluster since we were trying to read them as a pointer to an LVCluster which meant it was 8 bytes in size for 64 bit but that the assumption that the rawBytes would always be 8 byte aligned wouldn't hold good in that case. We should probably not make assumptions at all about byte alignment based on type and always have the user pass in the byte alignment requirements for the elements of the array.

I kept the above methods for now since they are used in a lot of places, We will fix them in a separate PR.

Reply:
The original code would have been more appropriate as `<LVCluster>` but that still doesn't work because a clusters alignment requirement can't be determined just from its size.  The templatized `bytes` is fine for any other type. You might be able to use static asserts or template specialization to make sure it LVCluster is not allowed.